dotnet/Simple.Sauce/SauceOptions.cs

```diff
@@ -13,18 +13,40 @@ public class SauceOptions
     {
         private const string DEFAULT_BROWSER_VERSION = "latest";
         private const string DEFAULT_PLATFORM = "Windows 10";
+        private string _buildName;
 
         public SauceOptions()
         {
-            WithChrome();
             Timeout = new Timeout();
+            SeleniumOptions = new ChromeOptions();
+        }
+
+        public SauceOptions(DriverOptions options)
+        {
+            SeleniumOptions = options;
+            if (options.BrowserName != null)
+            {
+                BrowserName = ToBrowserType(options.BrowserName);
+            }
         }
 
         public EdgeOptions ConfiguredEdgeOptions { get; set; } = new EdgeOptions();
         public ChromeOptions ConfiguredChromeOptions { get; private set; } = new ChromeOptions();
         public SafariOptions ConfiguredSafariOptions { get; set; } = new SafariOptions();
         public FirefoxOptions ConfiguredFirefoxOptions { get; set; } = new FirefoxOptions();
-        public Browser BrowserName { get; set; } = Browser.Chrome;
+        //public Browser BrowserName
+        //{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/SauceLabs/simple_sauce/pullRequest?prid=4945851)